components/speedreader/rust/lib/src/readability/src/scorer.rs

```diff
@@ -56,6 +57,43 @@ pub struct Candidate {
     pub score: Cell<f32>,
 }
 
+impl Candidate {
+    pub fn new(node: Rc<Node>, score: f32) -> Rc<Self> {
+        Rc::new(Candidate {
+            node: node,
+            score: Cell::new(score),
+        })
+    }
```

Comment:
I know I recommended you to use `Rc` here, but I didn't realize you already had a `Cell`. Fairly often (although not always) the combination of the two is indicative of code that works _around_ the borrow checker rather than with it. 

If this is required, I'd like to have some comment backing up _why_ - otherwise let's try to avoid it if possible.